code/addons/controls/package.json

```diff
@@ -32,31 +32,13 @@
       "require": "./dist/index.js",
       "import": "./dist/index.mjs"
     },
-    "./manager": {
-      "types": "./dist/manager.d.ts",
-      "require": "./dist/manager.js",
-      "import": "./dist/manager.mjs"
-    },
-    "./register": {
-      "types": "./dist/manager.d.ts",
-      "require": "./dist/manager.js",
-      "import": "./dist/manager.mjs"
-    },
+    "./manager": "./dist/manager.js",
+    "./register": "./dist/manager.js",
```

Comment:
manager and preview entries don't need types nor cjs outputs as they are always loaded in the manager or preview in the browser, never directly imported by the user.

Reply:
That's not true. My project was importing these types to allow us to overlay our own defaults. This change broke our workflow, it modified the exported output of the package and should not have been released under a minor tag as it changed the API.